b9lab-content/5-my-own-chain/03-starport.md

```diff
@@ -1 +1,198 @@
+
+Flags:
+  -h, --help   help for starport
+
+Use "starport [command] --help" for more information about a command.
+```
+
+There is a very good [introduction tutorial](https://docs.starport.network/guide/hello.html) provided by the [Starport documentation](https://docs.starport.network/) which you should first check and try to see the Hello-World response at [http://localhost:1317/cosmonaut/hello/hello/posts](http://localhost:1317/cosmonaut/hello/hello/posts). 
+
+## Your chain
+
+You can create a chain for Alice with:
+
+```bash
+$ starport scaffold chain github.com/alice/chain
+```
+
+The scaffolding will take some time. After the scaffolding you will have a folder `chain`. 
+In it you will see that we have 
+  - a folder for the application called `app`, 
+  - a folder for the commands called `cmd`, 
+  - a folder for the protobuff definitions called `proto`,
+  - a folder for the UI called `vue`, see [Vue.js](https://vuejs.org/).
+
+You can observe in the code following lines:
+
+```golang
+// this line is used by starport scaffolding # 1
+```
+
+so you should not replace those files in order to keep Starport working.
+
+So in the `chain` folder, run:
+
+```bash
+$ starport chain serve
+```
+
+The `starport chain serve` command will do several things:
+  - install all dependicies
+  - it will build protobuff files
+  - compile the application
+  - it will initialize the node with a single validator
+  - it will add accounts
+
+so at the end it will give you a testnet with a node running. Take a look at the `config.yml` file in the `chain` folder:
+
+```yaml
+accounts:
+  - name: alice
+    coins: ["20000token", "200000000stake"]
+  - name: bob
+    coins: ["10000token", "100000000stake"]
+validator:
+  name: alice
+  staked: "100000000stake"
+client:
+  vuex:
+    path: "vue/src/store"
+  openapi:
+    path: "docs/static/openapi.yml"
+faucet:
+  name: bob
+  coins: ["5token", "100000stake"]
+```
+
+in this file cou can set the accounts and the validator. In addition you can let Starport generate a client and a faucet. 
+
+You can observe the endpoints of the blockchain in the output of the `starport chain serve` command:
+
+```bash
+🌍 Tendermint node: http://0.0.0.0:26657
+🌍 Blockchain API: http://0.0.0.0:1317
+🌍 Token faucet: http://0.0.0.0:4500
+```
+
+If you make any change in the source, Starport will rebuild the binaries before starting the blockchain and keep the state. 
+
+Now, create a front end. Find the commands in the commands in the `readme.md` in the `chain` folder:
+
+```bash
+$ cd vue
+$ npm install
+$ npm run serve
+```
+
+Navigate to [http://localhost:8080/](http://localhost:8080/). See that no wallet is created or imported yet. Use the mnemonic for Alice in the output of the `starport chain serve` command to import a wallet.
+
+Now you can see Alices balance. In addition in the left bar, you can navigate to the custom type which will be empty at the moment. You can create a `message` with:
```

Comment:
```suggestion
Now you can see the balance of Alice's account. 

In the sidebar, select **Custom Type** to view the custom type. There are no custom types yet, so this page is empty at the moment. You can create a `message` with:
```

Reply:
let's write accessible content (avoid "left" and other positional statements)
Instead, follow https://developers.google.com/style/accessibility?hl=en 

here, the word "sidebar" is accurate 

(question: where do we see the account balance? not on the UI, so be sure to give instructions to view the account information in a file or with a command)